data/templates/default.html4

```diff
@@ -46,6 +46,12 @@ $endfor$
 $if(date)$
 <h3 class="date">$date$</h3>
 $endif$
+$if(abstract)$
+<div class="abstract">
+<p class="abstract">Abstract</p>
```

Comment:
I would remove this `<p>` element. After all, an abstract can contain multiple paragraphs, lists, etc.  If the abstract contains block-level content, it will be rendered with `<p>` tags and then you'll get invalid HTML with this setup.